libipl/libipl.H

```diff
@@ -43,6 +44,7 @@ const err_msg_map_type err_msg_map = {
 	{ IPL_ERR_FSI_TGT_NOT_FOUND, "FSI target is not available" },
 	{ IPL_ERR_FSI_REG, "FSI register read/write failure" },
 	{ IPL_ERR_PIB_TGT_NOT_FOUND, "PIB target is not available" },
+	{ IPL_ERR_CLK_FAILED, "Clock chip access is failed" },
```

Comment:
The clock is the chip in the Everest / Rainer? Just "Clock access is failed" is fine right? is it making sense?

Reply:
i think clock access will be a confusing term and it may not correctly implying that it failed when accessing clock generator chip.

To be more precice i think i have to use clock generator chip, instead of clock chip.

Reply:
it is clock hw generic failure. I will change the description